ddtrace/contrib/flask_cache/tracers.py

```diff
@@ -41,7 +55,7 @@ class TracedCache(Cache):
         _datadog_service = service
         _datadog_meta = meta
 
-        def __trace(self, cmd):
+        def __trace(self, cmd, write=False):
```

Comment:
Where is `write=False` used?

Reply:
Indeed it lacks documentation, but the redis backend of Flask-Cache was modified upstream to support multiple connections, one for writing and one for reading. This argument selects the right connection.

Reply:
is this comment stale @nizox? looks like the `write` argument isn't used in this function 🤔

Reply:
Good catch! I updated the PR.